lnrpc/invoicesrpc/invoices.proto

```diff
@@ -97,12 +97,29 @@ message AddHoldInvoiceRequest {
 }
 
 message AddHoldInvoiceResp {
+    bytes r_hash = 1;
```

Comment:
I don't understand why this is needed. In the `AddInvoice` call you can create an invoice by just giving the amount. There you don't know the `r_hash` in advance. But in this case you're giving the full payment request that contains the `r_hash` and `payment_addr`.
I think only the `add_index` should be added here.

Reply:
I don't think we need to return this since it's required to create a hold invoice? If we want to print on the CLI we can just use the hash provided in the request to re-iterate what the hash was.